services/src/handlers/tasks.rs

```diff
@@ -110,6 +114,51 @@ async fn list_handler<C: Context>(
     Ok(web::Json(task))
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct TaskAbortOptions {
+    #[serde(default)]
+    pub force: bool,
+}
+
+/// Retrieve the status of a task.
+///
+/// # Example
+///
+/// ## Request 1
+///
+/// ```text
+/// GET /tasks/420b06de-0a7e-45cb-9c1c-ea901b46ab69/abort
+/// Authorization: Bearer 4f0d02f9-68e8-46fb-9362-80f862b7db54
+/// ```
+///
+/// Response:
+///
+/// 200 OK
+///
+/// ## Request 2
+///  
+/// ```text
+/// GET /tasks/420b06de-0a7e-45cb-9c1c-ea901b46ab69/abort?force=true
```

Comment:
maybe explain the force parameter here again as part of the API doc

Reply:
[27eb0cc](https://github.com/geo-engine/geoengine/pull/586/commits/27eb0cc1d671a0468474157ac1309b1de8f164b8)